tests/functional/tests/test_charm.py

```diff
@@ -115,7 +115,7 @@ def add_machines(self, count: int, timeout: int = 600) -> None:
 
     @tenacity.retry(
         wait=tenacity.wait_fixed(5),
-        stop=tenacity.stop_after_attempt(12),
+        stop=tenacity.stop_after_attempt(24),
```

Comment:
**issue (non-blocking):** 

I hit timeout issue before with 12 attempts.

Reply:
That's why I used 24, so what is an issue here? Should I use more?

Reply:
Sorry I make something stupid here, I backwards the change. It's fine with 24.

Reply:
That's ok. I saw that you increase it, so I did same thing.